lib/car.rb

```diff
@@ -0,0 +1,19 @@
+class Car
+  attr_accessor :make, :model, :monthly_payment, :loan_length, :color
+  def initialize(make, model, monthly_payment, loan_length)
```

Comment:
I saw your comment. So yes, you are correct that you will need to separate out the `make` and `model`. So as it is passed, `Ford Mustang`, is there a method that will `split` a string?
So then you can either pass something like:
```
def initialize (make_model, monthly_payment, loan_length)
   @make = make_model.split(' ').first
   @model = make_model.split(' ').last
  ......
end
```
OR I did it like this:
```
attr_reader :make,
              :model,
              :monthly_payment,
              :loan_length,
              :color

  def initialize(car_type, monthly_payment, loan_length)
    @car_type        = car_type
    @make            = car_type_split.first
    @model           = car_type_split.last
    @monthly_payment = monthly_payment
    @loan_length     = loan_length
    @color           = nil
  end

  def car_type_split
    @car_type.split(" ")
  end
```